components/global/product_switcher_menu/product_switcher_menu.tsx

```diff
@@ -112,31 +112,33 @@ class ProductSwitcherMenu extends React.PureComponent<Props> {
                                 />
                             }
                         />
-                        <Menu.ItemExternalLink
-                            id='nativeAppLink'
-                            show={this.props.appDownloadLink && !UserAgent.isMobileApp()}
-                            url={useSafeUrl(this.props.appDownloadLink)}
-                            text={formatMessage({id: 'navbar_dropdown.nativeApps', defaultMessage: 'Download Apps'})}
-                            icon={
-                                <Icon
-                                    size={16}
-                                    glyph={'download-outline'}
-                                />
-                            }
-                        />
-                        <Menu.ItemToggleModalRedux
-                            id='about'
-                            modalId={ModalIdentifiers.ABOUT}
-                            dialogType={AboutBuildModal}
-                            text={formatMessage({id: 'navbar_dropdown.about', defaultMessage: 'About {appTitle}'}, {appTitle: this.props.siteName})}
-                            icon={
-                                <Icon
-                                    size={16}
-                                    glyph={'information-outline'}
-                                />
-                            }
-                        />
                     </TeamPermissionGate>
+                    <Menu.ItemExternalLink
+                        id='nativeAppLink'
+                        show={this.props.appDownloadLink && !UserAgent.isMobileApp()}
+                        url={useSafeUrl(this.props.appDownloadLink)}
+                        text={formatMessage({id: 'navbar_dropdown.nativeApps', defaultMessage: 'Download Apps'})}
+                        icon={
+                            <Icon
+                                size={16}
+                                glyph={'download-outline'}
+                            />
+                        }
+                        onClick={handleClick}
```

Comment:
This looks like a problem with a bad merge.

Reply:
Thanks for catching!